hazelcast/cluster.py

```diff
@@ -260,12 +286,14 @@ def _detect_membership_events(self, old, new):
 
     @staticmethod
     def _members_string(snapshot):
+        # type: (List[MemberInfo]) -> str
```

Comment:
As the name suggests, this is a MemberListSnapshot

Reply:
List[_MemberListSnapshot] or _MemberListSnapshot. I think List[_MemberListSnapshot] -> correct

Reply:
just the snapshot, this is not a list